.circleci/config.yml

```diff
@@ -1,46 +1,62 @@
-version: 2
+version: 2.1
+orbs:
+    gh: circleci/github-cli@1.0.3
 jobs:
-  build:
-    docker: 
-      - image: circleci/node:8.11-browsers
-    steps:
-      - checkout
-      - run: 
-          name: Install Dependencies
-          command: yarn
-      - run:
-          name: Verify Artifacts
-          command: yarn test
+    build:
+        docker:
+            - image: circleci/node:12.9.1-browsers
+        steps:
+            - checkout
+            - run:
+                  name: Install Dependencies
+                  command: yarn
+            - run:
+                  name: Verify Artifacts
+                  command: yarn test
 
-  publish:
-    docker:
-      - image: circleci/node:12.9.1-browsers
-    steps:
-      - checkout
-      - run:
-          name: Authenticate with registry
-          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
-      - run:
-          name: Publish @pxblue/mapbox
-          command:  |
-            MASTER_VERSION=`node -p "require('./package.json').version"`
-            NPM_LATEST_VERSION=`npm show @pxblue/mapbox version`
-            if ! [ $MASTER_VERSION == $NPM_LATEST_VERSION ];
-            then
-                npm publish
-            else
-                echo "Latest version is already published."
-            fi
+    publish:
+        docker:
+            - image: circleci/node:12.9.1-browsers
+        steps:
+            - checkout
+            - run:
+                name: Authenticate with registry
+                command: |
+                  echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
+                  echo "//registry.yarnpkg.com/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+            - run:
+                name: Publish @brightlayer-ui/mapbox
+                command: |
+                  yarn publish:package -b $CIRCLE_BRANCH
+
+    tag:
+        docker:
+            - image: circleci/node:12.9.1-browsers
+        steps:
+            - checkout
+            - gh/setup
+            - run:
+                name: Tag @brightlayer-ui/mapbox
+                command: |
+                  yarn tag:package -b $CIRCLE_BRANCH
```

Comment:
@emclaug2  do we need to use `yarn tag:package -b $CIRCLE_BRANCH -s -blui-mapbox` ?